int32 class_id
string label
float32 confidence
float32 xmin
float32 ymin
float32 xmax
float32 ymax
